pkg/liqoctl/common/cluster.go

```diff
@@ -620,22 +662,82 @@ func (c *Cluster) MapProxyIPForCluster(ctx context.Context, ipamClient ipam.Ipam
 	return nil
 }
 
+// UnmapProxyIPForCluster unmaps the ClusterIP address of the local proxy on the local external CIDR as seen by the remote cluster.
+func (c *Cluster) UnmapProxyIPForCluster(ctx context.Context, ipamClient ipam.IpamClient, remoteCluster *discoveryv1alpha1.ClusterIdentity) error {
+	clusterName := remoteCluster.ClusterName
+
+	// TODO: this logic will be moved on the Init function once
+	// the creation of the proxy deployment and service will be
+	// done at install time of liqo through the helm chart.
+
+	s, _ := c.printer.Spinner.Start(fmt.Sprintf("unmapping proxy ip for cluster {%s}", clusterName))
+
+	selector, err := metav1.LabelSelectorAsSelector(&liqolabels.ProxyServiceLabelSelector)
+	if err != nil {
+		s.Fail(fmt.Sprintf("an error occurred while retrieving proxy endpoint: %v", err))
+		return err
+	}
+	svc, err := liqogetters.GetServiceByLabel(ctx, c.locCtrlRunClient, c.namespace, selector)
+	if client.IgnoreNotFound(err) != nil {
+		s.Fail(fmt.Sprintf("an error occurred while retrieving proxy endpoint: %v", err))
+		return err
+	}
+	if k8serrors.IsNotFound(err) {
+		s.Warning(fmt.Sprintf("service for proxy not found, unable to unmap proxy ip for cluster {%s}", clusterName))
+		return nil
+	}
```

Comment:
the same here, I would use a switch to make clear that the conditions are mutually exclusive